app/code/Codifi/CustomerRequest/etc/frontend/di.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Checkout\Block\Cart\Item\Renderer">
+        <plugin name="setFalseValueToHasProductUrlFunction" type="Codifi\CustomerRequest\Plugin\Cart\Item\RendererPlugin" sortOrder="10"/>
```

Comment:
Not correct module (you should create a new one or move your plugin to a more appropriate module)